topic_statistics_demo/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(topic_statistics_demo)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rcutils)
+find_package(sensor_msgs REQUIRED)
+find_package(statistics_msgs REQUIRED)
```

Comment:
It's hard to know if you actually need these dependencies without an example already.

Reply:
Removed msg dependencies.